source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -78,13 +78,15 @@ package.
 
 .. code-block:: console
 
-   $ sudo mixin package add [package-name] [--build]
+   $ sudo mixin package add [package-name] [--bundle bundle-name] [--build]
 
 This command will add package-name to a bundle that is named after its parent
 repository. For example, if the RPM was provided locally, it will be added to
 the 'local' bundle. If it came from a repo that was added with :command:`mixin
-repo add` it will be added to a bundle named after the repo-name. The `--build`
-flag tells :command:`mixin` to run a `mixer` build after adding the package.
+repo add` it will be added to a bundle named after the repo-name. If the
```

Comment:
Please insert comma: "If it came from a repo that was added with :command:`mixin repo add`**,** ....."